src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm still evolving!\nWhat can I do for you, my highness?");
+        Scanner in = new Scanner(System.in);
+
+        ArrayList<Task> taskList = new ArrayList<Task>();
+
+        while (true) {
+            try {
+                System.out.println("Your highness:");
+                String input = in.nextLine();
+                String[] elem = input.split(" ");
+                if (input.equals("bye")) {
+                    System.out.println("Bye. Hope to see you again soon!");
+                    break;
+                } else if (input.equals("list")) {
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 1; i <= taskList.size(); i++) {
```

Comment:
Can do abstraction, instead of doing a for loop here, could have a printTaskList() method